Transfer/Sources/ScheduleTransferSectionData.swift

```diff
@@ -109,11 +112,18 @@ struct ScheduleTransferSummaryData: ScheduleTransferSectionData {
             let grossTransferAmount = transferAmountFormattedDouble + feeAmountFormattedDouble
 
             rows.append((title: "transfer_amount_confirmation".localized(),
-                         value: String(grossTransferAmount).format(with: destinationCurrency)))
-            rows.append((title: "transfer_fee_confirmation".localized(),
-                         value: feeAmount.format(with: destinationCurrency)))
+                         value: String(format: "%@ %@",
+                                       String(grossTransferAmount)
+                            .format(withActualDecimal: destinationCurrency), destinationCurrency)))
+            if feeAmount != "0.00" {
+                rows.append((title: "transfer_fee_confirmation".localized(),
```

Comment:
API returns formatted response so no need to format any value in this class. SO remove format from here. Just leave it as `String(format: "%@ %@",
                                       feeAmount,
                                       destinationCurrency))`

Reply:
Also remove extra ()